.github/workflows/main.yml

```diff
@@ -421,7 +416,8 @@ jobs:
       contents: read
       packages: write
     needs:
-      - build_rust_installer
+      - build_rayhunter
+      - build_rootshell
```

Comment:
we should make this change to the build_installer_gui_linux and build_installer_gui_windows jobs as well right?